src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -598,4 +599,11 @@ class SparqlQleverVisitor {
   void warnOrThrowIfUnboundVariables(auto* ctx,
                                      const SparqlExpressionPimpl& expression,
                                      std::string_view clauseName);
+
+  // Convert an instance of `Triples` to a `BasicGraphPattern` so it can be used
+  // just like a WHERE clause.
```

Comment:
Comment the odd cases (i.e. blank nodes vs. variables) as they are directly connected to the `CONSTRUCT WHERE` use case.